### PR TITLE
Adapt core library v4.0.0 breaking changes.

### PR DESCRIPTION
**Pull request by tarigo**
Summary of changes:
 - remove `LocalGlyphRasterizer`, it's now a part of core, pure C++ implementation;
 - LocationIndicatorLayer: deprecate `getImageTiltDisplacement` method, introduce it's `getImagePitchDisplacement` replacement;
 - the same as above for LocationPropertyFactory's imageTiltDisplacement methods;
 - change values in HeatmapLayerTest.testHeatmapRadiusAsConstant and SymbolLayerTest.testSymbolSpacingAsConstant
   tests to fit value limits for the style layer properties;

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.5.0-alpha.1 - September 3, 2020
```
Comment (Chaoba) [inline CHANGELOG.md:5]:
```suggestion
## 9.5.0-alpha.1 - September 4, 2020
```

File: MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java [hunk 2]
```diff
@@ -114,6 +114,19 @@ public void testImageTiltDisplacementAsConstant() {
...
+  @UiThreadTest
+  public void testImagePitchDisplacementAsConstant() {
+    Timber.i("image-tilt-displacement");
+    assertNotNull(layer);
+    assertNull(layer.getImagePitchDisplacement().getValue());
+
+    // Set and Get
+    Float propertyValue = 0.3f;
+    layer.setProperties(imageTiltDisplacement(propertyValue));
+    assertEquals(layer.getImagePitchDisplacement().getValue(), propertyValue);
```
Comment (Chaoba) [inline MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java:127]:
```suggestion
    assertEquals(propertyValue, layer.getImagePitchDisplacement().getValue());
```

Comment (Chaoba) [reply MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java:127]: Expected value should be the first param.

**Review by Chaoba — APPROVED** [review]
Others look good to me

File: CHANGELOG.md (hunk 1 above)
Comment (tarigo) [reply CHANGELOG.md:5]: Correct, but I would avoid to rebase - it'll breaks `android-internal`. :-)

File: MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java (hunk 2 above)
Comment (tarigo) [reply MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationIndicatorLayerTest.java:127]: :+1: Totally agree, we should fix all test cases in this test for consistency.

**Pull request merged**